src/api/utils/helpers.py

```diff
@@ -152,7 +152,7 @@ def find_item(data):
     ), 404)
 
 
-def response_builder(data, status_code):
+def response_builder(data, status_code=200):
```

Comment:
I think the status code should be explicitly stated by the developers who is building the response.

Reply:
Yes but most of the status_code will be 200, I don't think everyone should repeat that every time.